### PR TITLE
Add values for liveness and readiness probes in Query component.

### PR DESCRIPTION
**Pull request by ncastrocosta**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?
Add values for liveness and readiness probes in Query component.

### Checklist

- [X] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [X] User guide and development docs updated (if needed)

### CODE REVIEW

File: thanos/values.yaml [hunk 1]
```diff
@@ -380,6 +380,10 @@ query:
   # Pod affinity
   # Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#node-affinity
   affinity: {}
+  # set up store readinessProbe & livenessProbe
+  # Ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/
+  livenessProbe: {}
```
Comment (tarokkk) [inline thanos/values.yaml:385]:
```suggestion
  livenessProbe:
    httpGet:
      path: /-/healthy
      port: http
```

File: thanos/values.yaml [hunk 2]
```diff
@@ -380,6 +380,10 @@ query:
...
+  readinessProbe: {}
```
Comment (tarokkk) [inline thanos/values.yaml:386]:
```suggestion
  readinessProbe:
    httpGet:
      path: /-/ready
      port: http
```

**Review by tarokkk — CHANGES_REQUESTED** [review]
We shouldn't change the default behavior of the health checks. I added 2 suggestions. Other things looking good. Thanks

**Review by tarokkk — APPROVED** [review]

**Comment by ahma** [comment]
Thanks @ncastrocosta !

**Pull request merged**